common/src/main/scala/explore/components/Tile.scala

```diff
@@ -71,89 +67,72 @@ object Tile {
          (1024                           -> TileLGW)
     )
 
-  class Backend() {
-
-    val infoRef = Ref.toVdom[html.Element]
-
-    def render(p: Props, s: State) = {
-      val maximizeButton =
-        Button(
-          as = <.a,
-          basic = true,
-          compact = true,
-          clazz = ExploreStyles.TileStateButton |+| ExploreStyles.BlendedButton,
-          onClick = p
-            .sizeStateCallback(TileSizeState.Normal)
-            .when_(p.state === TileSizeState.Minimized) *> p
-            .sizeStateCallback(TileSizeState.Maximized)
-            .when_(p.state === TileSizeState.Normal)
-        )(Icons.Maximize)
-
-      val minimizeButton =
-        Button(
-          as = <.a,
-          basic = true,
-          compact = true,
-          clazz = ExploreStyles.TileStateButton |+| ExploreStyles.BlendedButton,
-          onClick = p
-            .sizeStateCallback(TileSizeState.Normal)
-            .when_(p.state === TileSizeState.Maximized) *> p
-            .sizeStateCallback(TileSizeState.Minimized)
-            .when_(p.state === TileSizeState.Normal)
-        )(Icons.Minimize)
-
-      <.div(ExploreStyles.Tile |+| ExploreStyles.FadeIn, p.key.whenDefined(^.key := _))(
-        <.div(
-          ExploreStyles.TileTitle,
-          p.back.map(b => <.div(ExploreStyles.TileButton, b)),
-          Menu(
+  val component =
+    ScalaFnComponent
+      .withHooks[Props]
+      // info ref
+      .useRefToVdom[html.Element]
+      .render { (p, infoRef) =>
```

Comment:
I tried to do `renderWithReuse` and I'm facing troubles to define `Reusability` for the ref

Reply:
A ref is a mutable holder (like cats-effect `Ref`). The ref is always the same, even though its `current` value may change. So I think it's safe to use `Reusability.always`.